examples/advanced/03_web_frameworks/fastapi_project/main.py

```
@@ -0,0 +1,392 @@
+# 헬스체크
+@app.get("/health")
+async def health_check():
+    """서버 상태 확인"""
+    try:
+        # API 키 확인
+        api_key_set = bool(os.getenv("OPENAI_API_KEY"))
+        
+        # 간단한 LLM 테스트
+        llm_working = False
+        try:
+            test_llm = get_llm("gpt-4o-mini")
```

Ruff (0.11.9)

Local variable `test_llm` is assigned to but never used

Remove assignment to unused variable `test_llm`

(F841)